app/components/goals-list/goals-list.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-const goalsList = props => {
+const goalsList = () => {
 
-  const goals = props.store.goals;
+  const goals = [];
```

Comment:
this should be coming from the state, but I've made it into an empty array for now as i don't know how to retrieve the actual goals